extensions/figma-files/package.json

```diff
@@ -28,34 +37,21 @@
       "interval": "2h"
     }
   ],
-  "preferences": [
-    {
-      "name": "PERSONAL_ACCESS_TOKEN",
-      "type": "password",
-      "required": true,
-      "title": "Personal Access Token",
-      "description": "Log in to Figma and click settings to setup a token"
-    },
```

Comment:
Let's keep this field and set the `required` field to `false` so that existing users who logged in through personal access tokens don't see the OAuth screen.